build.gradle

```diff
@@ -10,21 +10,21 @@ plugins {
 }
 
 ext {
-    wordPressUtilsVersion = '2.7.0'
-    wordPressLoginVersion = '0.19.0'
-    aztecVersion = 'v1.6.0'
+    wordPressUtilsVersion = '110-13b4f06680ed8c145cbd09d63c4998dfe8b3b9de'
+    wordPressLoginVersion = '97-b298ffaad9478bd0025833a01ee411f6d4ae6795'
+    aztecVersion = '1006-3fdbbc2086fb71d3750b01c013fa5cf6024c3142'
     gutenbergMobileVersion = 'v1.83.0'
```

Comment:
@AjeshRPai we could also test the changes from the editor by using the following reference `5184-8f82486da72f05160a4084716d89f45717fc4c00` in `gutenbergMobileVersion`. Let me know if you want me to commit the change or if you rather do it, thanks!

Reply:
Thanks, @fluiddot for providing me with the version number of the Gutenberg. I have committed the change in [5f07873](https://github.com/wordpress-mobile/WordPress-Android/pull/17153/commits/5f0787388a62b958d22ba954ed0f562c90edc7c8), Once again thanks for making the changes in Gutenberg. 👍🏼